### PR TITLE
Update some language tour dartpad embeds

### PR DESCRIPTION
**Pull request by parlough**

### CODE REVIEW

File: null_safety_examples/misc/lib/language_tour/callable_classes.dart [hunk 1]
```diff
@@ -7,4 +7,4 @@ class WannabeFunction {
 var wf = WannabeFunction();
 var out = wf('Hi', 'there,', 'gang');
 
-main() => print(out);
+void main() => print(out);
```
Comment (kwalrath) [inline null_safety_examples/misc/lib/language_tour/callable_classes.dart:10]: If we do this, can we remove the ignore_for_file comment?

File: src/_guides/language/language-tour.md [hunk 2]
```diff
@@ -1553,25 +1540,22 @@ prints a string that includes the value at the specified index.
...
-{{site.dartpad}}/5d70bc1889d055c7a18d35d77874af88
-{% endcomment %}
-
-<iframe
-src="{{site.dartpad-embed-inline}}?id=5d70bc1889d055c7a18d35d77874af88&split=60&ga_id=anonymous_functions"
-    width="100%"
-    height="400px"
-    style="border: 1px solid #ccc;">
-</iframe>
+```dart:run-dartpad:height-400px:ga_id-anonymous_functions:null_safety-true
```
Comment (kwalrath) [inline src/_guides/language/language-tour.md:1551]: Can we add a code-excerpt element above here?

**Review by kwalrath — APPROVED** [review]
The changes look good, thank you! I think we could tweak a couple of things.

File: null_safety_examples/misc/lib/language_tour/callable_classes.dart (hunk 1 above)
Comment (parlough) [reply null_safety_examples/misc/lib/language_tour/callable_classes.dart:10]: This is also used for the `wf` and `out` as they are considered public APIs. I can remove the ignore and type annotate those if desired though.

File: src/_guides/language/language-tour.md (hunk 2 above)
Comment (parlough) [reply src/_guides/language/language-tour.md:1552]: Good idea!

File: null_safety_examples/misc/lib/language_tour/callable_classes.dart (hunk 1 above)
Comment (kwalrath) [reply null_safety_examples/misc/lib/language_tour/callable_classes.dart:10]: Nope, that's fine. Never mind. :)

**Comment by kwalrath** [comment]
Would the 3 unusual build failures be fixed by running tool/dartformat.sh? (I didn't look too closely)

**Comment by parlough** [comment]
@kwalrath Yes! Thanks for pointing that out. I messed up the formatting when I fixed the merge conflict on GitHub. Should be good now.

**Pull request merged**